### DISCUSSION TITLE
Loot tracking weirdness

### DISCUSSION THREAD

**Discussion by Steiner-SE** [Lailloken/Exile-UI#254] [General]
I've been using the script for a bit now and other than the region related issue with tracking kills it's been working fine and tracking stuff for me.
As of today though it no longer tracks loot, at all, for some reason. I checked that the tracking was enabled, restarted the script and game, ran the permissions test, went as far as restarting the computer and reloading everything, still no go, it does no longer track my loot.

**Comment by Lailloken** [comment]
I just tested it, and it still works for me. Make sure the "stash" screen-check is still testing positive. There was a game update recently, so maybe the check has to be recalibrated.

**Reply by Steiner-SE** [reply]
Hmm, I'll check, I just ran the permission test. I was a bit tired so didn't think that far

**Comment by Steiner-SE** [comment]
Of course you were correct, as always. A quick recalibration sorted it. Thank you.
